app/src/pages/playground/InvocationParametersForm.tsx

```diff
@@ -16,113 +30,221 @@ const FormField = ({
+  field: InvocationParameter,
+  parameterInput: InvocationParameterInput
+): string | number | string[] | boolean | null | undefined => {
+  if (field.invocationInputField === undefined) {
+    throw new Error("Invocation input field is required");
+  }
+  return parameterInput[
+    toCamelCase(field.invocationInputField) as keyof InvocationParameterInput
+  ];
+};
+
+const makeInvocationParameterInput = (
+  field: InvocationParameter,
+  value: string | number | string[] | boolean | undefined
+): InvocationParameterInput | null => {
+  if (field.invocationName === undefined) {
+    throw new Error("Invocation name is required");
+  }
+  if (field.invocationInputField === undefined) {
+    throw new Error("Invocation input field is required");
+  }
+  return {
+    invocationName: field.invocationName,
+    canonicalName: field.canonicalName,
+    [toCamelCase(field.invocationInputField)]: value,
+  };
+};
 
 type InvocationParametersFormProps = {
-  model: ModelConfig;
-  onChange: InvocationParametersChangeHandler;
+  instance: PlaygroundInstance;
 };
 
 export const InvocationParametersForm = ({
-  model,
-  onChange,
+  instance,
 }: InvocationParametersFormProps) => {
-  const { invocationParameters, provider, modelName } = model;
-  // Get the schema for the incoming provider and model combination.
-  const schema = getInvocationParametersSchema({
-    modelProvider: provider,
-    modelName: modelName || "default",
-  });
+  const { model } = instance;
+  const updateInstanceModelInvocationParameters = usePlaygroundContext(
+    (state) => state.updateInstanceModelInvocationParameters
+  );
+  const filterInstanceModelInvocationParameters = usePlaygroundContext(
+    (state) => state.filterInstanceModelInvocationParameters
+  );
+  const { modelInvocationParameters } =
+    useLazyLoadQuery<InvocationParametersFormQuery>(
+      graphql`
+        query InvocationParametersFormQuery($input: ModelsInput!) {
+          modelInvocationParameters(input: $input) {
+            __typename
+            ... on InvocationParameterBase {
+              invocationName
+              label
+              required
+              canonicalName
+            }
+            ... on BoundedFloatInvocationParameter {
+              minValue
+              maxValue
+              invocationInputField
+              # defaultValueFloat: defaultValue
+            }
+            ... on IntInvocationParameter {
+              invocationInputField
+              # defaultValueInt: defaultValue
+            }
+            ... on StringInvocationParameter {
+              invocationInputField
+              # defaultValueString: defaultValue
+            }
+            ... on StringListInvocationParameter {
+              invocationInputField
+              # defaultValueStringList: defaultValue
+            }
+            ... on BooleanInvocationParameter {
+              invocationInputField
+              # defaultValueBool: defaultValue
```

Comment:
I think these default value fields have been removed from the schema

Reply:
the graphql schema? looks like their still there up above

Reply:
whoops, you are right